code/src/components/NotFound.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import notfound from '../images/notfound.png';
```

Comment:
Loved this photo! Maybe it could be centered on the page, though? Also, it seems a bit too big on screens above 1024px.